cmake/common/px4_base.cmake

```diff
@@ -511,6 +511,7 @@ function(px4_add_common_flags)
 		${PX4_BINARY_DIR}/src
 		${PX4_BINARY_DIR}/src/lib
 		${PX4_BINARY_DIR}/src/modules
+		${PX4_BINARY_DIR}/src/lib/FlightTasks
```

Comment:
Add comments that these are a temporary workaround and need to be fixed. PX4 builds without FlightTasks, ecl, matrix should be entirely possible.

Reply:
I will do that but just wanted to have the build fixed before tight things.